public/styles.css

```diff
@@ -450,6 +450,10 @@ select {
     cursor: pointer;
 }
 
+.asset-list tbody tr.focus-like {
+    background-color: rgb(225, 225, 234);
```

Comment:
It should be just .active or tr:focused I think it was. 
Must be in binary-style